.github/ISSUE_TEMPLATE/new-release.md

```diff
@@ -0,0 +1,76 @@
+---
```

Comment:
I think we don't have to port this release template as it is always taken from the main repository branch.